### PR TITLE
Deepcore

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request

Deepcore mining, why dig hole when machine dig hole for you?

## Why It's Good For The Game

Opens up the z-level to having more than rocks.

## Changelog
:cl:
code: sure has some of that
/:cl:

### CODE REVIEW

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```
Comment (Maldaris) [inline _maps/_basemap.dm:1]: This should get commented out again.

File: waspstation/code/game/area/dcm/ore_vein.dm [hunk 2]
```diff
@@ -4,49 +4,44 @@
...
 	material_rate_coeff = 0.3
 
 /area/lavaland/surface/ore_vein/uranium
 	name = "Uranium Ore Vein"
-	material_type = /datum/material/uranium
+	ore_type = /obj/item/stack/ore/uranium
 	material_rate_coeff = 0.2
 
 /area/lavaland/surface/ore_vein/diamond
 	name = "Diamond Ore Vein"
```
Comment (Maldaris) [inline waspstation/code/game/area/dcm/ore_vein.dm:77]: There's no such thing as "Diamond Ore", rather it's Kimberlite Ore which has diamond crystals. Consider making that change here.

**Review by Maldaris — COMMENTED** [review]
Couple small things, but there's a lot of noise in this PR, if we need to update the feature branch, lets do that.

**Comment by AsciiSquid** [comment]
dcmBare
Haha drill go brrrrrr

**Review by Maldaris — APPROVED** [review]

**Pull request merged**